src/theme/components/buttonGroup.ts

```diff
@@ -0,0 +1,10 @@
+const ButtonGroup = {
+  baseStyle: ({ buttonWidth }) => ({
+    width: buttonWidth === "default" ? "fit-content" : "100%",
+    button: {
+      flexGrow: buttonWidth === "default" ? "1" : "2",
```

Comment:
Why was this necessary?  This style applies to all buttons in the group, so the percentage by which each button will grow and shrink will use the same weighted value.  I pulled the branch locally to experiment.  When I changed to `flexGrow: “1”`, the buttons still sized properly in all cases.

Reply:
Ah, I see. This should be `flexGrow: buttonWidth === "default" ? undefined : "1",`. I wasn't sure which was the right one but the "1" for flexGrow makes it expands to the full width. This is how it should work, right?